src/test/regress/sql/multi_real_time_transaction.sql

```diff
@@ -0,0 +1,147 @@
+SET citus.next_shard_id TO 1610000;
```

Comment:
It might make sense to do a 
```
CREATE SCHEMA multi_real_time_transaction;
SET search_path TO 'multi_real_time_transaction';
...
DROP SCHEMA multi_real_time_transaction
```